test/integration/rtpbroadcast.js

```diff
@@ -34,7 +35,8 @@ describe('Rtpbroadcast tests', function() {
       .then(function(session) {
         janusSession = session;
         done();
```

Comment:
is it not better to use `.finally(done)` here or do we really want to catch exceptions?